director/director.go

```diff
@@ -35,14 +35,19 @@ type (
 	}
 
 	listServerResponse struct {
-		Name      string            `json:"name"`
-		AuthURL   string            `json:"authUrl"`
-		URL       string            `json:"url"`    // This is server's XRootD URL for file transfer
-		WebURL    string            `json:"webUrl"` // This is server's Web interface and API
-		Type      common.ServerType `json:"type"`
-		Latitude  float64           `json:"latitude"`
-		Longitude float64           `json:"longitude"`
-		Status    HealthTestStatus  `json:"status"`
+		Name               string            `json:"name"`
+		AuthURL            string            `json:"authUrl"`
+		BrokerURL          string            `json:"brokerUrl"`
+		URL                string            `json:"url"`    // This is server's XRootD URL for file transfer
+		WebURL             string            `json:"webUrl"` // This is server's Web interface and API
+		Type               common.ServerType `json:"type"`
+		Latitude           float64           `json:"latitude"`
+		Longitude          float64           `json:"longitude"`
+		EnableWrite        bool              `json:"enableWrite"`
+		EnableFallbackRead bool              `json:"enableFallbackRead"`
```

Comment:
Can you update these `Enable` variables to the new internal reps? They should be `EnableWrites` and `EnableDirectReads`, respectively. We should probably keep the json as it is, unless these are totally new and we don't need to worry about the wrong key being unmarshalled by an older server.